### PR TITLE
Re-add get-codeowners.ps1

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Re-Adding `get-codeowners.ps1` to unblock cpp. https://github.com/Azure/azure-sdk-for-cpp/blob/main/eng/pipelines/templates/stages/archetype-cpp-release.yml#L197

### CODE REVIEW

**Comment by benbp** [comment]
+1 assuming this is just a re-add as you say, but as a reviewer with less context it would be helpful to have a description of and/or links to the blocking issue, etc.

**Review by benbp — APPROVED** [review]

**Comment by chidozieononiwu** [comment]
> +1 assuming this is just a re-add as you say, but as a reviewer with less context it would be helpful to have a description of and/or links to the blocking issue, etc.
Its needed at https://github.com/Azure/azure-sdk-for-cpp/blob/main/eng/pipelines/templates/stages/archetype-cpp-release.yml#L197

File: eng/common/scripts/get-codeowners.ps1 [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  catch 
+  {
+    LogDebug "Invoke-RestMethod ${APIUrl} failed with exception:`n$_"
+    LogDebug "This might be because a team alias was used for user API request or vice versa."
+    return $false
+  }
+  return $true
+}
+
+nuget install "CodeOwnersParser" -OutputDirectory . `
```
Comment (weshaggard) [inline eng/common/scripts/get-codeowners.ps1:42]: @chidozieononiwu this doesn't look like the last committed state of this file. We never had a version that used the library like this. https://github.com/Azure/azure-sdk-tools/commit/4bfbe0075db68193a336ed4973df0505395069fd#diff-e3cfe80d53c2e2c455b3cc02c5592438031893d3e939b7abb57fefd94e734463 looks to be the last committed version.

**Review by weshaggard — CHANGES_REQUESTED** [review]
Lets add the last version of this unchanged..

**Review by weshaggard — APPROVED** [review]

File: eng/common/scripts/get-codeowners.ps1 (hunk 1 above)
Comment (chidozieononiwu) [reply eng/common/scripts/get-codeowners.ps1]: I have updated it.

**Pull request merged**